pyseir/load_data.py

```diff
@@ -639,7 +639,8 @@ def load_new_test_data_by_fips(fips, t0, smoothing_tau=5, correction_threshold=5
     df = df.loc[
         (df[CommonFields.POSITIVE_TESTS].notnull())
         & (df[CommonFields.NEGATIVE_TESTS].notnull())
-        & ((df[CommonFields.POSITIVE_TESTS] + df[CommonFields.NEGATIVE_TESTS]) > 0),
+        & (df[CommonFields.NEGATIVE_TESTS] > 0)
+        & (df[CommonFields.POSITIVE_TESTS] > 0),
```

Comment:
FWIW, positive tests == 0 is normal in some counties that have contained covid (or haven't been hit yet).

But maybe it's okay not to do any test correction in that case.

Reply:
The naming convention is confusing. Here CommonFields.POSITIVE_TESTS is cumulative. This is a very permissive filter.